### ISSUE TITLE
[HELP] "Failed to retrieve template child" for Rust 1.67.0-nightly

### ISSUE THREAD

**Issue by EnverOsmanov** [gtk-rs/gtk4-rs#1201]
Actions to reproduce:
1) Clone gtk4-rs repostiory
```
git clone git@github.com:gtk-rs/gtk4-rs.git
```
2) Navigate to the book's examples:
```
cd ./gtk4-rs/book/listings
```
3) Run an example which uses templates:
```
cargo run --color=always --package gtk4-rs-book-listings --bin composite_templates_1
```

Console's output:
>      Finished dev [unoptimized + debuginfo] target(s) in 0.03s
>      Running `target/debug/composite_templates_1`
> thread 'main' panicked at 'Failed to retrieve template child. Please check that it has been bound and has a #[template_child] attribute.', /home/enver/.cargo/registry/src/github.com-1ecc6299db9ec823/gtk4-0.5.2/src/subclass/widget.rs:1278:17
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by sdroege** [comment]
That's fixed by https://github.com/gtk-rs/gtk4-rs/pull/1192 . @bilelmoussaoui you didn't do a bugfix release since then?

**Comment by bilelmoussaoui** [comment]
Fixed in 0.5.3
